### DISCUSSION TITLE
cache decorator

### DISCUSSION THREAD

**Discussion by long76** [fastapi/fastapi#7429] [Questions] [answered]
### Is your feature request related to a problem

Need cache response on 30 seconds.

### The solution you would like

Like a [Flask-Caching](https://github.com/sh4nks/flask-caching).
```python
@cache.cached(timeout=86400, key_prefix='routes', hash_method=hashlib.sha3_256)
```

**Comment by tiangolo** [comment, marked as answer]
Check the conversation here: https://github.com/tiangolo/fastapi/issues/651

**Comment by long76** [comment]
@tiangolo thanks
